packages/terra-clinical-onset-picker/package.json

```diff
@@ -34,7 +34,7 @@
     "moment": "^2.22.2",
     "prop-types": "^15.5.8",
     "terra-breakpoints": "^2.0.0",
-    "terra-date-picker": "^3.0.0",
+    "terra-date-picker": "^4.0.0",
```

Comment:
This dependency bump should be included in the change log. Are there any non-passive changes to need to be tested for and considered with this change?

Reply:
Updated change log: https://github.com/cerner/terra-clinical/pull/571/commits/afb72e7ae5449b6df97f1b49a0114c491a298b36

terra-date-picker-4.0.0 [removed three props](https://github.com/cerner/terra-framework/blob/master/packages/terra-date-picker/CHANGELOG.md#breaking-changes) but [onset-picker](https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-onset-picker/src/OnsetPicker.jsx#L445-L454) doesn't use any of these props so there shouldn't be any impact. I tested it locally and didn't find any issues.